src/seccomp/src/lib.rs

```diff
@@ -1050,6 +1042,14 @@ impl SeccompFilter {
         }
         self
     }
+
+    /// Creates an empty SeccompFilter which allows everything
+    pub fn empty() -> SeccompFilter {
```

Comment:
What if you passed an `Option<SeccompFilter>` so that, when the user chooses to disable seccomp, the kernel really doesn't need to do any more work for each syscall (I assume the filter is composed of a few instructions, but still)?

Is there any hidden downside of passing empty filters to the kernel?

Another item - all the wording in the documentation refers to "seccomp *disabled*" when `seccomp-level=0`, this would change that into "seccomp enabled but doesn't do anything". All the docs should be changed.

Reply:
That's a good point. Maybe it would also make sense to encapsulate this inside SeccompFilter itself by making it an enum with two possibilities?

Reply:
I decided to test if the filter is empty and ignore the rest of the `SeccompFilter::apply` if it is.